Cargo.toml

```diff
@@ -8,6 +8,7 @@ authors = ["Keegan McAllister <mcallister.keegan@gmail.com>",
 description = "Fuzzing Rust code with american-fuzzy-lop"
 build = "build.rs"
 repostiory = "https://github.com/frewsxcv/afl.rs"
```

Comment:
Just realized I misspelled this, which explains why it doesn't show up in the crates.io listing